.config/ci/install.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+# Usage:
+# ./install.sh [install mode]
+
+# Detect install mode
+if [[ "${1}" == "libpcap" ]]; then
+    export SCAPY_USE_LIBPCAP="yes"
```

Comment:
I think that this does not work. The variable will be removed when `install.sh` stops.

Reply:
Shouldn't the `export` make sure it's available to other scripts?
In the case of github, we have the snippet below which uses a Github Actions functionality

Reply:
You're right.